components/BreadCrumbs.vue

```diff
@@ -31,15 +35,25 @@ export default {
 
     // filter invalid routes
     crumbLocations = crumbLocations.filter((location) => {
-      return (allRouteMap[location.name] && this.displayName(location, params));
+      return (allRouteMap[location.name]);
     });
 
     return {
-      crumbLocations, params, crumbPieces, allRouteMap
+      crumbLocations, params, crumbPieces, allRouteMap, clusterName: ''
     };
   },
 
   methods: {
+    async loadDeps() {
+      const clusterID = this.params.cluster;
+      const cluster = await this.$store.dispatch('management/find', { type: MANAGEMENT.CLUSTER, id: clusterID });
+
+      if (cluster.nameDisplay) {
+        this.clusterName = cluster.nameDisplay;
+      } else {
+        this.clusterName = clusterID;
```

Comment:
Keeping a copy of the name is making it non-reactive to changes instead of saving the cluster and using cluster.nameDisplay below.

(Also the cluster _should_ always be loaded before you get here so getById would probably be ok instead of async find)